pallets/elections/src/lib.rs

```diff
@@ -50,6 +46,16 @@ pub mod pallet {
     #[pallet::without_storage_info]
     pub struct Pallet<T>(_);
 
+    #[pallet::storage]
+    #[pallet::getter(fn members)]
+    pub type Members<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
+
+    #[pallet::storage]
+    pub type MembersPerSession<T> = StorageValue<_, u32, OptionQuery>;
```

Comment:
Does it need to be `Option` here? Is there any logic that depends or will depend in the future on `Option` state?

Reply:
changed to value